homeassistant/components/bmw_connected_drive/config_flow.py

```diff
@@ -70,7 +73,13 @@ async def async_step_user(
                 errors["base"] = "cannot_connect"
 
             if info:
-                return self.async_create_entry(title=info["title"], data=user_input)
+                return self.async_create_entry(
+                    title=info["title"],
+                    data={
+                        **user_input,
+                        CONF_REFRESH_TOKEN: info.get(CONF_REFRESH_TOKEN),
```

Comment:
Is it ok to set the refresh token to `None`?

This check is not checking for None:
https://github.com/home-assistant/core/blob/504f4a7acf6d3f5f7ba887b72671883a35a1feb8/homeassistant/components/bmw_connected_drive/coordinator.py#L40-L41

Reply:
Yes, the refresh token can be `None`. In that case, the library will use the normal login flow with username & password.